res/values-ro/strings.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" product="debug">Tastatură Unexpected (depanare)</string>
```

Comment:
The translation files must be formatted by running `python sync_translations.py`. (this helps maintaining them)

Reply:
Thanks! I probably missed that. I'll push an update in a few moments.